content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -77,6 +79,17 @@ about this format, see the [Kubernetes Protobuf serialization](https://github.co
 Interface Definition Language (IDL) files for each schema located in the Go
 packages that define the API objects.
 
+### OpenAPI V3
+
+{{< feature-state state="alpha"  for_k8s_version="v1.23" >}}
+
+OpenAPI V3 publication can be enabled with the feature gate `OpenAPIV3` and is disabled by default. See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/) for a genral explanation of feature gates and how to enable them.
+
+With the feature enabled, the Kubernetes API server serves an aggregated OpenAPI v3 spec per Kubernetes group version at the `/openapi/v3/apis/<group>/<version>` endpoint.
+The same request headers are supported in v3 and v2, with one exception that `application/com.github.proto-openapi.spec.v3@v1.0+protobuf` is used for the Accept header for protobuf OpenAPI v3.
```

Comment:
nit: it would be better to provide an amended table than to refer readers to review another table and then mentally patch it
(for beta and GA, this would be a must-fix rather than a nit)

Reply:
Also, I'd write the header name `Accept` in backticks.

Reply:
Updated with an amended table